index.js

```diff
@@ -5,34 +5,42 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-module.exports = {
-  extends: [
-    './rules/possible-errors',
-    './rules/best-practices',
-    './rules/strict-mode',
-    './rules/variables',
-    './rules/stylistic-issues',
-    './rules/es6',
-    './plugins/node/possible-errors',
-    './plugins/node/best-practices',
-    './plugins/node/stylistic-issues'
-  ].map(require.resolve),
-  parser: '@babel/eslint-parser',
-  env: {
-    browser: true,
-    es2020: true,
-    node: true
-  },
-  parserOptions: {
-    sourceType: 'module',
-    ecmaFeatures: {
-      // allow `return` statements in the global scope
-      globalReturn: false,
-      // enable global strict mode
-      impliedStrict: true,
-      // enable JSX
-      jsx: true
+import babelParser from '@babel/eslint-parser';
+import eslintLayoutFormatting from './rules/layout-formatting.js';
+import eslintPossibleProblems from './rules/possible-problems.js';
+import eslintSuggestions from './rules/suggestions.js';
+import globals from 'globals';
+import nodePlugin from './plugins/node.js';
+
+export default [
+  eslintLayoutFormatting,
+  eslintPossibleProblems,
+  eslintSuggestions,
+  nodePlugin,
+  {
+    languageOptions: {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
the need for all these `eslint-disable` is unfortunate. why does eslint flag what appears to be standard config options?

Reply:
I agree. The situation is that we're running all configs, including typescript, against the JavaScript in this repo – which is not typescript – as a test. Probably best if I turn these off in the local `eslint.config.js`. It still tests the validity of the configs.

Reply:
Updated for review.